cmd/printers/table/collector.go

```diff
@@ -182,8 +183,17 @@ func (r *ResourceStateCollector) processEvent(ev event.Event) error {
 // update for a resource.
 func (r *ResourceStateCollector) processStatusEvent(e event.StatusEvent) {
 	if e.Type == event.StatusEventResourceUpdate {
+		klog.V(7).Infoln("processing status event")
 		resource := e.Resource
-		previous := r.resourceInfos[resource.Identifier]
+		if resource == nil {
```

Comment:
So making sure it doesn't crash is good, but do we know why this happened in the first place? The idea is that the `StatusEventResourceUpdate` is about a specific resource, so why doesn't the event include the resource?

Reply:
I don't think this is happening in this bug. But we should program defensively.